### PR TITLE
VZE: Implement mobility device flag (crash) & asmp road rating (location)

### LINKED ISSUE #306: VZE: Implement mobility device flag (crash) & asmp road rating (location)
both should be editable.

continues work from #190 & #191

### LINKED ISSUE #388: VZE: Page bombs when you click "edit polygon" from Location details page
Screen Shot 2019-10-31 at 3 41 05 PM

### PR DESCRIPTION
**Pull request by mddilley**
Closes #306 
Closes #388

### CODE REVIEW

**Comment by mddilley** [comment]
This PR adds editable micromobility flag to the Crash view and editable ASMP road ranking to the Location view (along with a new asmp lookup table in Hasura staging, `atd_txdot__asmp_level_lkp` ). I also updated `react-map-gl` to the latest release to address #388.

Since editing was not implemented in the Location view yet and in order to avoid repetitive code, I refactored the data display tables that are used in Crash and Location views as `<DataTable />`. This component uses the parameters set in `crashDataMap.js` or `locationDataMap.js` to render `<Card />` components for each `title` and all `fields` defined in these data maps (editable and non-editable). `<DataTable />` can handle nested data (using the reference format for `lodash.get()`), dollar format, date format, and data returned as strings from GraphQL queries.

### Micromobility flag
Screen Shot 2019-11-04 at 10 36 14 AM

### ASMP Level
Screen Shot 2019-11-04 at 10 35 55 AM

File: atd-vze/src/queries/lookups.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { gql } from "apollo-boost";
+
+export const GET_LOOKUPS = gql`
```
Comment (mateoclarke) [inline atd-vze/src/queries/lookups.js]: 👍

File: atd-vze/src/views/Locations/locationDataMap.js [hunk 2]
```diff
@@ -4,18 +4,32 @@ const locationDataMap = [
...
       last_update: {
         label: "Last Update",
+        editable: false,
         format: "datetime",
       },
       est_comp_cost: {
         label: "Estimated Comprehensive Cost",
+        // If data is nested in returned data object, define path in
+        //  lodash.get() format to expose in DataTable component
+        dataPath: ["crashes_count_cost_summary", "est_comp_cost"],
```
Comment (mateoclarke) [inline atd-vze/src/views/Locations/locationDataMap.js]: good stuff!

**Comment by mddilley** [comment]
Thanks, @mateoclarke! Hopefully, this will reduce maintenance on these two views.

**Pull request merged**